docs/monitoring.md

```diff
@@ -69,36 +83,53 @@ follows:
   </tr>
 </table>
 
+### Spark configuration options
+
 <table class="table">
   <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
   <tr>
     <td>spark.history.provider</td>
-    <td>org.apache.spark.deploy.history.FsHistoryProvider</td>
+    <td><code>org.apache.spark.deploy.history.FsHistoryProvider</code></td>
     <td>Name of the class implementing the application history backend. Currently there is only
     one implementation, provided by Spark, which looks for application logs stored in the
     file system.</td>
   </tr>
   <tr>
+    <td>spark.history.retainedApplications</td>
+    <td>50</td>
```

Comment:
any particular reason to reorder this option?  I'm biased to keeping the old ordering just for git history